BDArmory/Control/MissileFire.cs

```diff
@@ -6166,6 +6175,12 @@ public void SendTargetDataToMissile(MissileBase ml, bool clearHeat = true)
                         if (BDArmorySettings.DEBUG_MISSILES)
                             Debug.Log($"[BDArmory.MissileData]: targetInfo sent for {ml.targetVessel.Vessel.GetName()}");
                     }
+                    if (ml.hasDataLink && vesselRadarData)
+                    {
+                        ml.vrd = vesselRadarData;
+                        ml.radarTarget = TargetSignatureData.DataLinkNoTarget(ml.transform.position, ml.maxStaticLaunchRange, ml.activeRadarRange);
+                        if (BDArmorySettings.DEBUG_MISSILES) Debug.Log("[BDArmory.MissileData]: Waiting for DataLink");
+                    }
```

Comment:
Unnecessary. VRD already handles receiving data from a linked radar in the Missilebase radar targeting code; why are you essentially duplicating this?